client/src/common/helpers.js

```diff
@@ -120,6 +120,45 @@ export const setStateProp = (prop, data, options) => {
   return obj;
 };
 
+export const viewPropsChanged = (nextViewOptions, currentViewOptions) => {
```

Comment:
Optional: naming update...  it is what it says... but maybe instead of "view" something like ```checkViewPropsChanged``` or ```verify...```